source/_posts/potential-patent-infringement.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How to check whether your product causes potential patent infringement
+date: 2020-07-24 18:16:09
+author: Abhinav Arora
+category:
+  - ["IP-Patents"]
+tags:
```

Comment:
Keyword   should be linked with website,

Reply:
Added two keywords